packages/cpp/src/browser/cpp-build-configurations.spec.ts

```diff
@@ -20,15 +20,15 @@ import { FileSystem } from "@theia/filesystem/lib/common";
 import { StorageService } from "@theia/core/lib/browser/storage-service";
 import { MockStorageService } from "@theia/core/lib/browser/test/mock-storage-service";
 import sinon = require("sinon");
-import { CppBuildConfigurationManager, CppBuildConfiguration } from "./cpp-build-configurations";
+import { CppBuildConfigurationManager, CppBuildConfiguration, CppBuildConfigurationChanger } from "./cpp-build-configurations";
 import { FileSystemNode } from "@theia/filesystem/lib/node/node-filesystem";
 import { bindCppPreferences } from "./cpp-preferences";
 import { PreferenceService } from "@theia/core/lib/browser/preferences/preference-service";
 import { MockPreferenceService } from "@theia/core/lib/browser/preferences/test/mock-preference-service";
 
 let container: Container;
 
-beforeEach(function() {
+beforeEach(function () {
```

Comment:
Can you exclude these unrelated whitespace changes?  I know it's annoying, these are due to a difference between how vscode and typescript-language-server (used by theia) format it:

https://github.com/theia-ide/theia/issues/1789